File: Assets/Scripts/Result/Result.js
#pragma strict

import SimpleJSON;
import UnityEngine.UI;

function Start () {
	var data : TextAsset = Resources.Load.<TextAsset>("Stories");
	var json : JSONNode = JSON.Parse(data.text);
	var stories : JSONArray = json["Stories"].AsArray;
	
	for(var i=0;i<stories.Count;i++) {
		PlayerPrefs.SetString("Result", PlayerPrefs.GetString("Result")+PlayerPrefs.GetString("Q"+i));
	}
	
	GetComponent(Text).text = PlayerPrefs.GetString("Result");
}

File: Assets/Scripts/Result/Result.cs
using UnityEngine.UI;
using UnityEngine;
using System.Collections;

[System.Serializable]
public partial class Result : MonoBehaviour
{
    public virtual void Start()
    {
        int i = 0;
        while (i < Global.stories.Count)
        {
            PlayerPrefs.SetString("Result", PlayerPrefs.GetString("Result") + PlayerPrefs.GetString("Q" + i));
            i++;
        }
        ((Text) this.GetComponent(typeof(Text))).text = PlayerPrefs.GetString("Result");
    }

}